.github/workflows/databases-nightly.yml

```diff
@@ -0,0 +1,53 @@
+name: databases-nightly
+
+on:
+  pull_request: # TODO: Remove before merge
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+
+ mariadb:
+    runs-on: ubuntu-22.04
+
+    services:
+      mariadb:
+        image: quay.io/mariadb-foundation/mariadb-devel:verylatest
+        env:
+          MARIADB_ALLOW_EMPTY_ROOT_PASSWORD: yes
+          MARIADB_DATABASE: laravel
+        ports:
+          - 3306:3306
+        options: --health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    strategy:
+      fail-fast: true
+
+    name: MariaDB Very Latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 8.2
+          extensions: dom, curl, libxml, mbstring, zip, pcntl, pdo, pdo_mysql, :php-psr
+          tools: composer:v2
+          coverage: none
+
+      - name: Set Framework version
+        run: composer config version "11.x-dev"
```

Comment:
Why is this needed?

Reply:
I copied that from the other workflows. Not sure if it is needed.

Reply:
Ah I remember now. All good.